templates/download/server/thank-you.html

```diff
@@ -44,96 +44,7 @@ <h1 style="font-weight: 100;">Thank you for downloading Ubuntu Server</h1>
   </div>
 </div>
 
-{% include "download/shared/_get_ebook_v1.html" with return_url="https://www.ubuntu.com/download/server/thank-you" %}
-
-<!-- dependencies -->
-<script src="{{ ASSET_SERVER_URL }}ec520d10-XMLHttpRequest.min.js"></script>
-<script src="{{ ASSET_SERVER_URL }}4176b39e-serialize.js"></script>
-<script src="{{ ASSET_SERVER_URL }}6b7597df-event-listener-polyfill.js"></script>
-<!-- script -->
-<script>
-
-  /**
-   * Given a form element, extract its data and its "action" target URL
-   * and submit the form using Ajax
-   */
-  backgroundSubmit = function(formElement, submitCallback) {
```

Comment:
OMG is background submit dead?! That's wonderful news!

Reply:
no, I just moved it